st2common/st2common/models/api/rbac.py

```diff
@@ -31,7 +34,7 @@
 
 
 class RoleAPI(BaseAPI):
-    model = PackDB
```

Comment:
How this this work before? Doesn't seem to me that the PackDB model resembles roles at all